### PR TITLE
Pyepics

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: recipes/pyepics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
+      - python
+      - setuptools
+      - numpy
```
Comment (synapticarbors) [inline recipes/pyepics/meta.yaml]: Is `numpy` required to `build`?

File: recipes/pyepics/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  run:
+      - python
+      - numpy
```
Comment (synapticarbors) [inline recipes/pyepics/meta.yaml]: it looks like this package also uses `pyparsing`. It's optional, so it's up to you if it should be a req. That said, so `numpy` is also optional.

File: recipes/pyepics/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+test:
+  imports:
+    - epics
```
Comment (synapticarbors) [inline recipes/pyepics/meta.yaml]: Add other import tests:

```
    - epics.autosave
    - epics.compat
    - epics.devices
```

File: recipes/pyepics/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+about:
+  home: http://pyepics.github.io/pyepics/
+  license: Epics Open
+  summary: Python interface to Epics Channel Access
```
Comment (synapticarbors) [inline recipes/pyepics/meta.yaml]: `dev_url: https://github.com/pyepics/pyepics`
`doc_url: http://pyepics.github.io/pyepics`

File: recipes/pyepics/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      - python
+      - numpy
+
+test:
+  imports:
+    - epics
+
+about:
+  home: http://pyepics.github.io/pyepics/
+  license: Epics Open
```
Comment (synapticarbors) [inline recipes/pyepics/meta.yaml]: I think you need to manually package the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Please copy the url to this PR as a comment in the `meta.yaml`:

https://github.com/pyepics/pyepics/pull/108

**Comment by CJ-Wright** [comment]
Thank you very much!

**Pull request merged**